test/onnx/test_pytorch_onnx_caffe2.py

```diff
@@ -898,6 +940,7 @@ def test_cosine_similarity(self):
         self.run_model_test(torch.nn.CosineSimilarity(dim=1, eps=1e-6), train=False,
                             input=(x, y), batch_size=BATCH_SIZE, use_gpu=False)
 
+    @skipIfUnsupportedOpsetVersion([10])
```

Comment:
why this failed on opset 10?

Reply:
LSTM exports with Slice ops (Slice exports with all inputs in opset 10 rather than one input and attributes, which is not the case with the caffe2 op)

Reply:
I see, make sense. I would suggest to add this context into the comments later.